client/web/reporter/src/components/Common/Prefilter.vue

```diff
@@ -1,5 +1,7 @@
 <template>
-  <c-form-table-wrapper class="prefilter">
+  <c-form-table-wrapper
+    hide-add-button
```

Comment:
In general, if the component has only one prop, it should be in one line.
But no biggie